packages/calcite-components/src/components/input-time-picker/input-time-picker.scss

```diff
@@ -8,14 +8,23 @@
 
 :host([scale="s"]) {
   --calcite-toggle-spacing: theme("spacing.2");
+  --calcite-internal-input-text-input-padding-inline-end: calc(
+    var(--calcite-toggle-spacing) + theme("spacing.4")
+  ); // chevron gap + icon spacing
```

Comment:
Are these comments necessary? Wonder if another internal variable might help here instead?

Reply:
I took another stab at this with Sass functions. I'm trying to avoid creating another internal var that would probably be lengthy and not be minified in the output build.